### PR TITLE
feat: Integration tests

### PR DESCRIPTION
**Pull request by Raffy23**
## This PR
- Adds integration simple tests:

| Test                                                | keptn-0.14.2-prometheus-15.10.1 | keptn-0.15.1-prometheus-15.10.1 | keptn-0.16.0-prometheus-15.10.1 |
| :-------------------------------------------------- | :-----------------------------: | :-----------------------------: | :-----------------------------: |
| TestPodtatoheadEvaluation/Configure_Prometheus      |       :heavy_check_mark:        |       :heavy_check_mark:        |       :heavy_check_mark:        |
| TestPodtatoheadEvaluation/Deploy_podtatohead_v0.1.1 |       :heavy_check_mark:        |       :heavy_check_mark:        |       :heavy_check_mark:        |
| TestPodtatoheadEvaluation/Deploy_podtatohead_v0.1.2 |       :heavy_check_mark:        |       :heavy_check_mark:        |       :heavy_check_mark:        |
| TestPodtatoheadEvaluation                           |       :heavy_check_mark:        |       :heavy_check_mark:        |       :heavy_check_mark:        |

*(From run: https://github.com/Raffy23/prometheus-service/actions/runs/2607617483)*

### CODE REVIEW

File: test/events/podtatohead.configure-monitoring.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "contenttype": "application/json",
+  "data": {
+    "project": "e2e-project",
+    "service": "podtatoserver",
+    "temporaryData": {
+      "distributor": {
+        "subscriptionID": ""
+      }
+    },
```
Comment (christian-kreuzberger-dtx) [inline test/events/podtatohead.configure-monitoring.json:10]: Btw you only need those data if you directly talk to prometheus-service. If you're sending this event to Keptn api, please remove `temporaryData`

File: test/events/podtatohead.deploy-v0.1.2.triggered.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "data": {
+    "configurationChange": {
+      "values": {
+        "image": "ghcr.io/j-poecher/podtato-head/podtatoserver:v0.1.2"
```
Comment (christian-kreuzberger-dtx) [inline test/events/podtatohead.deploy-v0.1.2.triggered.json:5]: Do we need the image from j-poecher, or can we use the official one, e.g.:
```
IMAGE="ghcr.io/podtato-head/podtatoserver"
VERSION=v0.1.1
SLOW_VERSION=v0.1.2
```

File: test/events/podtatohead.deploy-v0.1.1.triggered.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "data": {
+    "configurationChange": {
+      "values": {
+        "image": "ghcr.io/j-poecher/podtato-head/podtatoserver:v0.1.1"
+      }
+    },
```
Comment (christian-kreuzberger-dtx) [inline test/events/podtatohead.deploy-v0.1.1.triggered.json:7]: same here, can we use the official image, e.g.
```
IMAGE="ghcr.io/podtato-head/podtatoserver"
VERSION=v0.1.1
SLOW_VERSION=v0.1.2
```

File: test/events/podtatohead.deploy-v0.1.2.triggered.json (hunk 2 above)
Comment (Raffy23) [reply test/events/podtatohead.deploy-v0.1.2.triggered.json]: It seems to work with the official image, I replaced it

File: test/e2e/podtatohead_test.go [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+		{FilePath: "../data/podtatoserver-0.1.0.tgz", ResourceName: fmt.Sprintf("charts/%s.tgz", testEnv.EventData.Service)},
+		{FilePath: "../data/locust.basic.py", ResourceName: "locust/basic.py"},
+		{FilePath: "../data/locust.conf", ResourceName: "locust/locust.conf"},
+		{FilePath: "../data/podtatohead.sli.yaml", ResourceName: "prometheus/sli.yaml"},
+		{FilePath: "../data/podtatohead.slo.yaml", ResourceName: "slo.yaml"},
+		{FilePath: "../data/podtatohead.remediation.yaml", ResourceName: "remediation.yaml"},
+	}
+
+	err = testEnv.SetupTestEnvironment()
+	//	require.NoError(t, err)
```
Comment (TannerGabriel) [inline test/e2e/podtatohead_test.go:43]: Are these lines still relevant?

**Review by TannerGabriel — APPROVED** [review]
LGTM

File: test/e2e/podtatohead_test.go (hunk 4 above)
Comment (Raffy23) [reply test/e2e/podtatohead_test.go:43]: Hmm this seems to be an oversight from my part, that code shouldn't be commented out ...

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**